custom/download_qrda.php

```diff
@@ -150,10 +150,10 @@ function closeMe() {
         <th scope="col" class="multiDownload">
             <input type="checkbox" name="checkAll" id="checkAll"/>
             <div style="display:none" id=downloadAll>
-                <img class='downloadIcon' src='<?php echo $GLOBALS['webroot'];?>/images/downbtn.gif' onclick=downloadAllXML(); />
+                <img class="downloadIcon" src="<?php echo $GLOBALS['onclick=downloadAllXML()'];?>"/>
```

Comment:
this doesn't look right

Reply:
You're right, my bad. On another note, I searched for that function and couldn't find where it's defined or used anywhere else. Should I remove the onClick entirely? I fixed the image issue.